js/src/Ui.js

```diff
@@ -1,6 +1,7 @@
 import Mediaplayer from "./Mediaplayer.js";
 import NoopMediaplayer from "./NoopMediaplayer.js";
 import ScaledImageTexture from "./ScaledImageTexture.js";
+import SparkMediaplayer from "wpe-lightning-spark/src/platforms/spark/SparkMediaplayer.mjs";
```

Comment:
without this explicit import SparkMediaplayer was not know in this scope.

Reply:
Let's import the SparkMediaPlayer only if `Utils.isSpark`.